src/main/kotlin/org/rust/ide/annotator/RustHighlightingAnnotator.kt

```diff
@@ -39,14 +52,30 @@ class RustHighlightingAnnotator : Annotator {
             }
         }
 
-        override fun visitPath(o: RustPathElement) {
-            o.reference.resolve().let {
-                if (it is RustPatBindingElement && it.isMut) {
-                    holder.highlight(o.identifier, RustColor.MUT_BINDING)
-                }
+        // @TODO ReferenceElement
```

Comment:
I had trouble replacing RustPathElement with RustReferenceElement, but didn't look into it